packages/markdown-support/src/load-plugins.ts

```diff
@@ -9,7 +9,7 @@ async function importPlugin(p: string | UnifiedPluginImport): UnifiedPluginImpor
   return await p;
 }
 
-export function loadPlugins(items: Plugin[]): Promise<[unified.Plugin] | [unified.Plugin, unified.Settings]>[] {
+export function loadPlugins(items: Plugin[]): Promise<[unified.Plugin] | [unified.Plugin, any[]]>[] {
```

Comment:
There's no `unified.Settings` anymore (see the [types rewrite in `unified@10`](https://github.com/unifiedjs/unified/commit/b3e2464adf2e2bd667446aee572d7c870c3b9019#diff-093ad82a25aee498b11febf1cdcb6546e4d223ffcb49ed69cc275ac27ce0ccceL229-L231))

The [release notes](https://github.com/unifiedjs/unified/releases/tag/10.0.0) state:

> Use `any[]` instead of `[Record<string, unknown>?]` for the default plugin type parameters

So I've changed all instances of `unified.Settings` to `any[]`.

Reply:
Huh, interesting change on their part!

Reply:
It changed from `[Settings]` to `any[]`. There could be zero or more parameters passed to plugins, and not just limited to objects (`unified.Settings` was a confusing name for `Record<string, unknown>`).
So I think either the types here should be `any`, or perhaps the code solved to allow more than one parameter — but those aren’t used much nor recommended

Reply:
Switched to `any` instead of `any[]` as suggested